go.mod

```diff
@@ -1,9 +1,13 @@
 module github.com/ONSdigital/dp-cantabular-dimension-api
 
-go 1.17
+go 1.19
 
 replace github.com/coreos/etcd => github.com/coreos/etcd v3.3.24+incompatible
 
+replace github.com/prometheus/client_golang => github.com/prometheus/client_golang v1.11.1
+
+replace golang.org/x/text => golang.org/x/text v0.3.8
```

Comment:
I think we wouldn't need this directive if we ran `go get golang.org/x/text@latest`?

Reply:
I tried what you suggested and that broke the audit in CI

So, i've had to put it back.

Reply:
Now fixed